ansible_builder/main.py

```diff
@@ -96,35 +223,57 @@ def __init__(self, definition,
         self.build_steps()
 
     def build_steps(self):
-        self.steps = []
-        self.steps.append("FROM {}".format(self.base_image))
-        self.steps.append(self.newline_char)
-        [self.steps.append(step) for step in GalaxySteps(containerfile=self)]
+        self.steps = [
+            "FROM {}".format(self.base_image),
+            ""
+        ]
+        self.steps.extend(
+            GalaxySteps(containerfile=self)
+        )
 
         return self.steps
 
     def write(self):
         with open(self.path, 'w') as f:
             for step in self.steps:
-                if step == self.newline_char:
-                    f.write(step)
-                else:
-                    f.write(step + self.newline_char)
+                f.write(step + self.newline_char)
 
         return True
 
 
 class GalaxySteps:
-    def __new__(cls, *args, containerfile):
+    def __new__(cls, containerfile):
         definition = containerfile.definition
-        if not definition.galaxy_requirements_file:
-            return []
-        src = definition.galaxy_requirements_file
-        dest = containerfile.build_context
-        copy(src, dest)
-        basename = os.path.basename(definition.galaxy_requirements_file)
-        return [
-            "ADD {} /build/".format(basename),
-            "RUN ansible-galaxy role install -r /build/{} --roles-path /usr/share/ansible/roles".format(basename),
-            "RUN ansible-galaxy collection install -r /build/{} --collections-path /usr/share/ansible/collections".format(basename)
-        ]
+        steps = []
+        if definition.python_requirements_file:
+            f = definition.python_requirements_file
+            f_name = os.path.basename(f)
+            steps.append(
+                "ADD {} /build/".format(f_name)
+            )
+            shutil.copy(f, containerfile.build_context)
+            steps.extend([
+                "",
+                "RUN pip3 install -r {0}".format(f_name)
```

Comment:
I still need to provide some examples of the output, in test directories or otherwise.